llm/gpt-oss-sft/README.md

```diff
@@ -0,0 +1,128 @@
+# Finetuning OpenAI gpt-oss Models with SkyPilot
+
+![](https://i.imgur.com/TkoqCQK.png)
+
+On August 5, 2025, OpenAI released [gpt-oss](https://openai.com/open-models/), including two state-of-the-art open-weight language models: `gpt-oss-120b` and `gpt-oss-20b`. These models deliver strong real-world performance at low cost and are available under the flexible Apache 2.0 license.
+
+The `gpt-oss-120b` model achieves near-parity with OpenAI o4-mini on core reasoning benchmarks, while the `gpt-oss-20b` model delivers similar results to OpenAI o3-mini.
+
+This guide walks through how to finetune both models with LoRA/full finetuning using [🤗 Accelerate](https://github.com/huggingface/accelerate).
+
+![Cloud Logos](https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/cloud-logos-dark.png)
+
+## Step 0: Setup infrastructure
+
+SkyPilot is a framework for running AI and batch workloads on any infrastructure, offering unified execution, high cost savings, and high GPU availability.
+
+### Install SkyPilot
+
+```bash
+pip install 'skypilot[all]'
+```
+For more details on how to setup your cloud credentials see [SkyPilot docs](https://docs.skypilot.co).
+
+### Choose your infrastructure
+
+```bash
+sky check
+```
+
+## Step 1: Run gpt-oss models
+
+### Full finetuning
+
+**For `gpt-oss-20b` (smaller model):**
+- Requirements: 1 node, 8x H100 GPUs
+```bash
+sky launch -c gpt-oss-20b-sft gpt-oss-20b-sft.yaml
+```
+
+**For `gpt-oss-120b` (larger model):**
+- Requirements: 4 nodes, 8x H200 GPUs each
+```bash
+sky launch -c gpt-oss-120b-sft gpt-oss-120b-sft.yaml
+```
+
```

Comment:
Can we inline the yaml here for better visibility?

Reply:
Updated